pkg/client/k8s.go

```diff
@@ -743,3 +743,19 @@ func (k K8sClient) GetPods(labelSelector string, fieldSelector string, namespace
 	}
 	return pods, nil
 }
+
+/*
+	LOCAL VOLUMES SPECIFIC METHODS
+*/
+
+// GetLocalPvDeployment returns the deployment-list with a specific
+// label selector query
+func (k K8sClient) GetLocalPvDeployment(labelSelector string) (*appsv1.DeploymentList, error) {
```

Comment:
this function can be renamed to `GetDeployments` instead, do update the name in the comment too.